cmd/zpool/zpool_main.c

```diff
@@ -2389,6 +2392,9 @@ zpool_do_import(int argc, char **argv)
 				mntopts = optarg;
 			}
 			break;
+		case 'p':
+			flags |= ZFS_IMPORT_SCRUB_PAUSE;
+			break;
```

Comment:
I'm fine with dropping the `import -p` option. My use case was the same - to address #3658